.stylelintrc.json

```diff
@@ -5,12 +5,16 @@
         "header/header": ["config/license.js", {}],
         "length-zero-no-unit": [true, { "ignore": "custom-properties" }],
         "selector-type-no-unknown": [true, { "ignore": ["custom-elements"] }],
+        "selector-not-notation": "complex",
         "selector-pseudo-element-colon-notation": ["single", {}],
         "custom-property-pattern": "^_?([a-z][a-z0-9]*)(-[a-z0-9]+)*$",
         "no-duplicate-selectors": null,
         "selector-class-pattern": null,
         "no-descending-specificity": null,
-        "declaration-block-no-redundant-longhand-properties": null
+        "declaration-block-no-redundant-longhand-properties": null,
+        "declaration-empty-line-before": null,
+        "import-notation": null,
```

Comment:
would we not want to pick one style over the other? Whats the added benefit to having both? this comment is for line 15-16

Reply:
I don't understand the question. This is deactivating the opinionated stylelint setting that is aggressively changing the Spectrum CSS output in unexpected ways. It just preserves the provided content instead of altering it more.

Reply:
thank you for this context. it looks like it mirrors the settings in CSS.